config/enabled.yml

```diff
@@ -91,6 +91,10 @@ Style/BracesAroundHashParameters:
   Description: 'Enforce braces style around hash parameters.'
   Enabled: true
 
+Style/CaseConditionRepeat:
```

Comment:
This should be a lint cop. I'd change the name to "DuplicateCaseCondition" or "DuplicateCaseBranch".